gcp/appengine/blog/content/posts/announcing-guided-remediation-in-osv-scanner/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Announcing Guided Remediation in OSV-Scanner"
+date: 2024-03-26T00:00:00Z
+draft: false
+author: Michael Kedar
+---
+
+Addressing vulnerabilities in project dependencies can often be overwhelming for software developers. OSV-Scanner's new [Guided Remediation feature](https://google.github.io/osv-scanner/experimental/guided-remediation/) aims to simplify this process by prioritizing and fixing the vulnerabilities that matter most in your projects.
+<!--more-->
+The tool is now available with the OSV-Scanner 1.7.0 release under the `osv-scanner fix` subcommand. Currently, it only supports npm `package.json` / `package-lock.json` dependencies, but we're planning to expand support to more ecosystems in the future.
+
+
+## What can Guided Remediation do?
+Guided Remediation scans your project's manifest and lockfile, identifying vulnerable dependencies and suggesting upgrades that will resolve them. There are two different upgrade strategies to choose from:
+- In-place lockfile modification, which does targetted updates to vulnerable packages while still maintaining compatibility with the existing constraints in your dependency graph.
+- Relocking and relaxing requirements, to install the most up-to-date versions of your dependencies and upgrade your direct requirements to remove vulnerable packages from the dependency graph.
+
+In conjunction with these strategies, the tool also allows you to filter vulnerabilities based on metrics like CVSS severity, dependency depth, and whether they affect production or development dependencies. This allows you to control which vulnerabilities you wish to target.
```

Comment:
We should add a  point on prioritizing by return on  investment. i.e.  helping prioritise which direct  upgrades to do first by  #vulns resolved.

Reply:
I've tried adding a sentence conveying this.